### ISSUE TITLE
[Legacy] [master] Most likely missing type

### ISSUE THREAD

**Issue by dilijev** [Microsoft/ChakraCore#1538]
https://github.com/Microsoft/ChakraCore/wiki/Build-Status#legacy-builds

Broken since http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/master/job/daily_dev12_x64_debug/143/

See build log:
http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/master/job/daily_dev12_x64_debug/143/consoleFull#-447890337a82fefab-f698-416f-8fca-58544c94cd4e

```
WScriptJsrt.cpp(728): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int [d:\j\workspace\daily_dev12_x---55cdd258\bin\ch\ch.vcxproj]
10:58:18 WScriptJsrt.cpp(728): error C2146: syntax error : missing ',' before identifier 'size' [d:\j\workspace\daily_dev12_x---55cdd258\bin\ch\ch.vcxproj]
...
```

See http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/master/job/daily_dev12_x64_debug/143/changes for commits between the last good build and the current broken build.

To repro locally please follow instructions here:
https://github.com/Microsoft/ChakraCore/wiki/Jenkins-Repro-Steps#legacy-builds-windows-only

When submitting a fix, remember to request legacy tests:
https://github.com/Microsoft/ChakraCore/wiki/Jenkins-Build-Triggers#legacy-builds

**Comment by dilijev** [comment]
Tagging authors of the commits between last known good and the first failure: @antonmes @obastemur @Fishrock123 @agarwal-sandeep @mrkmarron @LouisLaf @EdMaurer @xanlpz 

Can you all verify whether the break was caused by your changes?

**Comment by Fishrock123** [comment]
probably not

**Comment by dilijev** [comment]
Assigning @ThomsonTan to assess this issue.

**Comment by obastemur** [comment]
Should be fixed by #1584 ?
